### PR TITLE
Polkadot Protocol > Architecture > System Chains > Overview

### PR DESCRIPTION
**Pull request by CrackTheCode016**
Migrates the https://wiki.polkadot.network/docs/learn-system-chains Wiki page

### CODE REVIEW

**Comment by CrackTheCode016** [comment]
@eshaben @dawnkelly09 Do you think replacing the table with just the headings, with "Encointer"  being a sub-section under collections, as this is the only exception for Kusama. WDYT?

**Comment by eshaben** [comment]
> @eshaben @dawnkelly09 Do you think replacing the table with just the headings, with "Encointer" being a sub-section under collections, as this is the only exception for Kusama. WDYT?

Yeah, I think the tabs get lost between the headers. I'd move it out of the tabbed element. It might be nice to have a mermaid diagram that has a Polkadot element and a Kusama element that list the supported system chains for each, so users can quickly glance at it and understand what system chains belong to what ecosystem. Then yeah have the following sections go over each system chain

**Review by dawnkelly09 — COMMENTED** [review]
Made updates for grammar & formatting. Tested links and updated where needed.  Requesting final formatting look from Erin.

File: polkadot-protocol/architecture/system-chains/overview.md [hunk 1]
```diff
@@ -1,4 +1,102 @@
...
+```
+
+All system parachains are on both Polkadot and Kusama with the following exceptions:
+
+- [**Collectives**](#collectives) - only on Polkadot
+- [**Encointer**](#encointer) - only on Kusama
+
+### Asset Hub
+
+The [Asset Hub](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus#asset-hub-){target=\_blank} is an asset portal for the entire network. It helps asset creators, such as reserve-backed stablecoin issuers, track the total issuance of an asset in the network, including amounts transferred to other parachains. It is also the point where they can use transactions like mint and burn to manage the on-chain asset.
```
Comment (eshaben) [inline polkadot-protocol/architecture/system-chains/overview.md:56]:
```suggestion
The [Asset Hub](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus#asset-hub-){target=\_blank} is an asset portal for the entire network. It helps asset creators, such as reserve-backed stablecoin issuers, track the total issuance of an asset in the network, including amounts transferred to other parachains. It also serves as the hub where asset creators can perform on-chain operations, such as minting and burning, to manage their assets effectively.
```

File: polkadot-protocol/architecture/system-chains/overview.md [hunk 2]
```diff
@@ -1,4 +1,102 @@
...
+
+This asset management logic is encoded directly in the runtime of the chain rather than in smart contracts. The efficiency of executing logic in a parachain allows for fees and deposits that are about 1/10th of what is required on the relay chain. These low fees mean that the Asset Hub is well suited for handling the frequent transactions required when managing balances, transfers, and on-chain assets.
+
+The Asset Hub also supports non-fungible assets (NFTs) via the [Uniques pallet](https://polkadot.js.org/docs/substrate/extrinsics#uniques){target=\_blank} and [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts){target=\_blank}. For more information about NFTs, see the Polkadot Wiki section on [NFT Pallets](https://wiki.polkadot.network/docs/learn-nft-pallets){target=\_blank}.
```
Comment (eshaben) [inline polkadot-protocol/architecture/system-chains/overview.md:60]:
```suggestion
The Asset Hub also supports non-fungible assets (NFTs) via the [Uniques pallet](https://polkadot.js.org/docs/substrate/extrinsics#uniques){target=\_blank} and [NFTs pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts){target=\_blank}. For more information about NFTs, see the Polkadot Wiki section on [NFT Pallets](https://wiki.polkadot.network/docs/learn-nft-pallets){target=\_blank}.
```

Are there better links for this than Polkadot.js - like a Rust docs link or something?

File: polkadot-protocol/architecture/system-chains/overview.md [hunk 3]
```diff
@@ -1,4 +1,102 @@
...
+
+Participants are requested to attend physical key-signing ceremonies with small groups of random people at randomized locations. These local meetings are part of one global signing ceremony occurring at the same time. Participants use the Encointer wallet app to participate in these ceremonies and manage local community currencies. 
+
+Referendums marking key Encointer adoption milestones include: 
+
+- [Referendum 158 - Register Encointer As a Common Good Chain](https://kusama.polkassembly.io/referendum/158){target=\_blank} registered Encointer as the second system parachain on Kusama's network
+- [Referendum 187 - Encointer Runtime Upgrade to Full Functionality](https://kusama.polkassembly.io/referendum/187){target=\_blank} - introduced a runtime upgrade bringing governance and full functionality for communities to use the protocol
```
Comment (eshaben) [inline polkadot-protocol/architecture/system-chains/overview.md:98]:
```suggestion
- [**Referendum 158 - Register Encointer As a Common Good Chain**](https://kusama.polkassembly.io/referendum/158){target=\_blank} - registered Encointer as the second system parachain on Kusama's network
- [**Referendum 187 - Encointer Runtime Upgrade to Full Functionality**](https://kusama.polkassembly.io/referendum/187){target=\_blank} - introduced a runtime upgrade bringing governance and full functionality for communities to use the protocol
```

**Review by eshaben — COMMENTED** [review]
Nice! Looks much better

**Review by dawnkelly09 — APPROVED** [review]
🚀

**Review by nhussein11 — APPROVED** [review]
niceeeee! LGTM!

**Pull request merged**